### DISCUSSION TITLE
문제 지문 가져오는 API가 있을까요?

### DISCUSSION THREAD

**Discussion by tjrkdgnl** [solvedac/unofficial-documentation#43] [질답] [answered]
- Doc를 보니 없을 것 같은데.. 혹시 API가 있을까요?
- 현재 백준에서는 웹 스크래핑을 금지하고 있기 때문에 크롤링은 진행할 수 없을 것 같아서 질문드립니다 ㅜㅜ

**Comment by RanolP** [comment, marked as answer]
제가 알기론 없습니다. (https://github.com/solvedac/unofficial-documentation/discussions/34#discussioncomment-1700980 참조)
